### DISCUSSION TITLE
DAI 2023-2024 - Practical work 1 - Stefani Massimo & Troeltsch Jonas

### DISCUSSION THREAD

**Discussion by massteff** [heig-vd-dai-course/heig-vd-dai-course#90] [Show and tell]
Our practical work 1 will be a command-line interface (CLI) capable of performing text translations using an AI, specifically DeepL.

**Comment by ludelafo** [comment]
Great, thank you!

**Comment by jonas-tr** [comment]
The repo : https://github.com/JonasTroeltsch/translation-cli
The last hash : 98a45c04586ee7ebade562cdaad0a8d32b8e0dad
